.github/workflows/commit.yaml

```diff
@@ -76,6 +76,15 @@ jobs:
           go-version: ${{ matrix.go-version }}
           cache: true
 
+      # Ensure the pagefile is large enough to execute tests like TestStore_hammer_close on Windows.
+      - name: configure Pagefile
+        uses: al-cheb/configure-pagefile-action@v1.2
```

Comment:
@evacchi I noticed latest version is 1.3, are we hard-coding a version because of a problem?